app/dataSources/errorMsgs.tsx

```diff
@@ -0,0 +1,9 @@
+const UNAUTHENTICATED_ERROR_MESSAGE = "You must be signed in to continue. Please log in to access this feature.";
+const UNAUTHORIZED_ERROR_MESSAGE = "You don't have permission to do this. Please reach out to your league administrator for help.";
```

Comment:
You are fine with these  being called `UNAUTHENTICATED_` & `UNAUTHORIZED_` despite those not being what the http status code short name's are?

Reply:
I followed the naming convention of the API, for consistency. I'm ok. Are you ok with this?